Makefile

```diff
@@ -65,7 +65,7 @@ DB_FQDN=db.unstable.shuttle.rs
 CONTAINER_REGISTRY=public.ecr.aws/shuttle-dev
 DD_ENV=unstable
 USE_TLS?=disable
-RUST_LOG?=shuttle=trace,debug
+RUST_LOG?=shuttle=debug,info
```

Comment:
I think staging should still have traces if we can somehow figure out how to reduce the ambulance task